aim/sdk/utils.py

```diff
@@ -84,3 +84,56 @@ def check_types_compatibility(dtype: str, base_dtype: str, update_base_dtype_fn=
     if dtype == 'list' and any_list_regex.match(base_dtype):
         return True
     return False
+
+
+def get_installed_packages():
```

Comment:
what about moving all these util functions into `aim/ext` module? looks to be completely independent from sdk stuff.

Reply:
yeah moving to `aim/ext/resources/utils.py` is more convenient